### DISCUSSION TITLE
Email Results

### DISCUSSION THREAD

**Discussion by Dan-krm** [ditdot-dev/vue-flow-form#211] [Q&A] [answered]
How to send the results to my email?

**Comment by spinn** [comment, marked as answer]
@Dan-krm You'd need some kind of back-end for this (Python, Node.js, PHP, ...). Or maybe look into Mandrill, Sendgrid, Mailgun and similar email APIs, they might have some kind of JavaScript library you can directly use.

**Comment by yuvi100** [comment]
I used EmailJS and just implemented it inside the **`onSendData()`**  function after **`const data = this.getData()`** like so:
        
        emailjs.send('YOUR_SERVICE_ID', 'YOUR_TEMPLATE_ID', data, 'YOUR_USER_ID')
            .then(function(response) {
              console.log('SUCCESS!', response.status, response.text);
            }, function(error) {
              console.log('FAILED...', error);
            });

There's more info on these parameters [here](https://www.emailjs.com/docs/sdk/send/).

Don't forget to import EmailJS first `import emailjs from 'emailjs-com'` and also set up your free account [here](https://www.emailjs.com/) so that you can create your email template and assign the data that comes across from the form submission.

This might not be the perfect solution but I hope this helps.

**Reply by adambouthillette** [reply]
Hey. Where did you put your import emailjs statement? Just wondering because that line specifically causes my program to crash. Thanks

**Reply by yuvi100** [reply]
In this section of the .vue file:

```
  // Import necessary components and classes
  import FlowForm from '../../src/components/FlowForm.vue'
  import QuestionModel, { QuestionType, ChoiceOption, LinkOption } from '../../src/models/QuestionModel'
  import LanguageModel from '../../src/models/LanguageModel'
  import emailjs from 'emailjs-com'
```

Also I'm not aware of what JS experience you have, but you need to install EmailJS by running the following command in your terminal in your project folder: `npm install emailjs-com`
